frontend/src/metabase/internal/lib/components-webpack.js

```diff
@@ -22,4 +24,16 @@ const components = [
   ...getComponents(containersReq),
 ];
 
+// provide some stats on the total vs total documented components
+const documented = getComponents(componentsReq).length;
+
+// get everything and then subtract documented
+const total = getComponents(allComponentsReq).length - documented;
+
+export const stats = {
+  total,
+  documented,
```

Comment:
It looks like `total` and `documented` aren't used outside of computing `ratio`, so I'd probably not export them.

Reply:
I ended up using them to get a bit more introspection.